### ISSUE TITLE
Is the project dead?

### ISSUE THREAD

**Issue by XLPE** [alibaba/ApsaraCache#15]
I haven't seen new release for long time, so would be great to know the plans with this.
Thanks

**Comment by kingpeterpaule** [comment]
no, we will PR new feature soon. Bacause we are very busy this period.


------------------------------------------------------------------
发件人：XLPE <notifications@github.com>
发送时间：2018年11月12日(星期一) 14:52
收件人：alibaba/ApsaraCache <ApsaraCache@noreply.github.com>
抄　送：Subscribed <subscribed@noreply.github.com>
主　题：[alibaba/ApsaraCache] Is the project dead? (#15)

I haven't seen new release for long time, so would be great to know the plans with this.
 Thanks

**Comment by wangwj** [comment]
一贯作风
